envoy/api/v2/cds.proto

```diff
@@ -356,6 +356,9 @@ message Cluster {
     // .. note::
     //   The specified percent will be truncated to the nearest 1%.
     core.Percent healthy_panic_threshold = 1;
+    //Configuration for ZoneAwareLoadBalancer
+    core.Percent zone_aware_routing_enabled = 100;
```

Comment:
proto fields should be in order (2 and 3). I think you are mixing up proto field numbers with defaults.

Reply:
Got it . Sorry for such silly mistake .